docs/monitoring/metrics.md

```diff
@@ -436,6 +436,35 @@ metrics.reporter.stsd.port: 8125
 
 {% endhighlight %}
 
+### Datadog (org.apache.flink.metrics.datadog.DatadogHttpReporter)
+
+In order to use this reporter you must copy `/opt/flink-metrics-datadog-{{site.version}}.jar` into the `/lib` folder
+of your Flink distribution.
+
+Parameters:
+
+- `apikey` - the Datadog API key
+- `tags` - (optional) the global tags that will be applied to metrics when sending to Datadog. Tags should be separated by comma only
+
+Example configuration:
+
+{% highlight yaml %}
+
+metrics.reporters: dghttp
+metrics.reporter.dghttp.class: org.apache.flink.metrics.datadog.DatadogHttpReporter
+metrics.reporter.dghttp.apikey: xxx
+metrics.reporter.dghttp.tags: myflinkapp,prod
+
+// <host>, <job_name>, <tm_id>, <subtask_index>, <task_name>, <operator_name> will be sent to Datadog as tags
+metrics.scope.jm: <host>.jobmanager
```

Comment:
simplified it and moved out of code highlight.

I believe it's necessary to explain to users how to find and filter tags in Datadog, in order to ramp them up faster and remove any potential friction.